internal/checker/cloudformation/rules/encryption/verify_rds_not_publicly_accessible.rego

```diff
@@ -0,0 +1,48 @@
+# METADATA
+# title: "Verify that public access to RDS data is prohibited."
+# description: "This policy guarantees the protection of all your public AWS Application Load Balancers from application-layer attacks by integrating them with the Web Application Firewall (AWS WAF) service, ensuring that they remain inaccessible to the public."
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbinstance.html
+# custom:
+#   id: CB_CFT_008
+#   severity: MEDIUM
+package lib.cloudformation.CB_CFT_008
+import future.keywords.in
+
+resource := "AWS::RDS::DBInstance"
+
+is_valid{
+    some resources in input.Resources
+    resources.Type == resource
+}
+
```

Comment:
PubliclyAccessible is not required value means default value is always false
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbinstance.html#cfn-rds-dbinstance-publiclyaccessible

swap logic for fail and pass and return resources since we want to return an object {}
fail[resources] {
    is_valid
    some resources in input.Resources
    resources.Properties.PubliclyAccessible == true
}